### PR TITLE
Refactored variable names and method syntax

### PR DESCRIPTION
**Pull request by PeregrineReed**
I made a few adjustments to game stats, mostly in the class, but I had to make a couple tweaks to the test as well.

### CODE REVIEW

**Comment by zachlp2016** [comment]
Interesting.  I like it Peregrine.

File: lib/game_stats.rb [hunk 1]
```diff
@@ -123,15 +116,13 @@ def average_goals_by_season
   end
 
   def average_game_goals
-    average = total_scores.sum / @games.size.to_f
-    average.round(2)
+    (total_scores.sum / @games.size.to_f).round(2)
   end
 
-  def wins_percentage(team)
+  def wins_percentage(site)
```
Comment (nagerz) [inline lib/game_stats.rb]: What is site?

Comment (PeregrineReed) [reply lib/game_stats.rb]: This one was tricky to name for sure, I googled what the status of home vs away is called and found no common name, only suggestions for site, berth, status, etc. Host implies home in my opinion. Team_status crossed my mind too, but felt close to team_stats.

**Comment by nagerz** [comment]
Still would like to look at that name but going to merge

**Pull request merged**